phylogenetic/config/229e/dropped_strains.txt

```diff
@@ -1,3 +1,4 @@
 OK662398.1
 OK625404.1
 MZ712010.1
+PP810610.1
```

Comment:
FWIW, it's been extremely helpful to see comments on _why_ a sample is being excluded, e.g. [zika's exclude file](https://github.com/nextstrain/zika/blob/24f9f3fd7ec8f81a29a06d49ab74a038bba1b41f/phylogenetic/defaults/dropped_strains.txt)

Reply:
> FWIW, it's been extremely helpful to see comments on _why_ a sample is being excluded, e.g. [zika's exclude file](https://github.com/nextstrain/zika/blob/24f9f3fd7ec8f81a29a06d49ab74a038bba1b41f/phylogenetic/defaults/dropped_strains.txt)

updated.